package.json

```diff
@@ -119,6 +119,7 @@
     "start:bee": "yarn start -- examples/agents/bee.ts",
     "docs:build": "embedme --source-root=. docs/**/*.md && cp *.md docs/ && yarn lint:fix docs/ && yarn prettier --write docs/",
     "docs:check": "yarn docs:build --verify",
+    "docs:links": "linkinator \"**/*.md\" --markdown --directory-listing --resurse --skip node_modules --skip \"https://ai.meta.com/blog/meta-llama-3-1/\" ",
```

Comment:
Please explain.

Reply:
I should have mentioned that `https://ai.meta.com/blog/meta-llama-3-1/` is skipped because it is flagged as failing with a 400, but I could not see anything wrong with the link and I could access it ok in my browser.

I think that's likely what you were asking about, most of the other options I think reflect the name of the option with --markdown saying to check markdown files, and --directory-listing asking for directory info

Reply:
I removed resurse as it was a type and obviously not needed

Reply:
- Yes, but it seems like we could leverage the functionality of the target library to check HTML.
- I would put this script under `docs:check` script

Regarding the error. I assume you are getting it because the crawler is detected as bot, but nevermind.

Reply:
> I would put this script under docs:check script

Do you mean add `&& yarn docs:links` to docs:check or integrate the contents of the `docs:links` target directly into `docs:checks` ?


> Yes, but it seems like we could leverage the functionality of the target library to check HTML.

@Tomas2D I can take a look to also include HTML files, are those just html files that are in the repo itself versus being generated from markdown or something like that (I've not looked to see if there is any automation/generation of a website).

I'm out all next week so it won't be until after I get back that I could look at the HTML side of things. Also happy to add in a follow on PR.

Reply:
I added the `yarn docs:links` to the `docs:checks` target, let me know if that was not what you wanted.

Reply:
There is just one single HTML file (`index.html`) that loads Docsify, which converts Markdown documents into HTML. Hence, everything is done by the client (browser).

Reply:
Feel free to do so in the next PR. It is just an improvement; there is, so there's no need to rush.